src/lib/day-calendar/day-calendar.service.ts

```diff
@@ -51,25 +51,26 @@ export class DayCalendarService {
   }
 
   generateMonthArray(config: IDayCalendarConfigInternal, month: Moment, selected: Moment[]): IDay[][] {
+    const parsedMonth = month.isValid() ? month.clone() : moment();
```

Comment:
Can we cover it with a test? e2e or unit

Reply:
Added a unit test ;-)